File: Inventario/Web_Farmacia/Clases/Asignacion.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace Web_Farmacia.Clases
{
    public class Asignacion
    {
        private int id_asignacion;
        private int id_usuario;
        private int id_personal;
        private string personal;
        private int id_area;
        private string area;
        private int id_gerencia;
        private string gerencia;
        private int id_bienes;
        private string bien;
        private string codigo;
        private int id_inventario;
        private string inventario;
        private DateTime fecha_asig;

        public int Id_asignacion
        {
            get
            {
                return id_asignacion;
            }

            set
            {
                id_asignacion = value;
            }
        }

        public int Id_usuario
        {
            get
            {
                return id_usuario;
            }

            set
            {
                id_usuario = value;
            }
        }

        public int Id_inventario
        {
            get
            {
                return id_inventario;
            }

            set
            {
                id_inventario = value;
            }
        }

        public int Id_bienes
        {
            get
            {
                return id_bienes;
            }

            set
            {
                id_bienes = value;
            }
        }

        public string Bien
        {
            get
            {
                return bien;
            }

            set
            {
                bien = value;
            }
        }

        public string Codigo
        {
            get
            {
                return codigo;
            }

            set
            {
                codigo = value;
            }
        }

        public string Inventario
        {
            get
            {
                return inventario;
            }

            set
            {
                inventario = value;
            }
        }

        public int Id_personal
        {
            get
            {
                return id_personal;
            }

            set
            {
                id_personal = value;
            }
        }

        public string Personal
        {
            get
            {
                return personal;
            }

            set
            {
                personal = value;
            }
        }

        public int Id_area
        {
            get
            {
                return id_area;
            }

            set
            {
                id_area = value;
            }
        }

        public string Area
        {
            get
            {
                return area;
            }

            set
            {
                area = value;
            }
        }

        public int Id_gerencia
        {
            get
            {
                return id_gerencia;
            }

            set
            {
                id_gerencia = value;
            }
        }

        public string Gerencia
        {
            get
            {
                return gerencia;
            }

            set
            {
                gerencia = value;
            }
        }

        public DateTime Fecha_asig
        {
            get
            {
                return fecha_asig;
            }

            set
            {
                fecha_asig = value;
            }
        }
    }
}

File: App_Movil/Proyecto_Inventario/app/src/main/java/com/example/proyecto_inventario/ui/Consultar_Asignacion/Clases/Asignacion.java
package com.example.proyecto_inventario.ui.Consultar_Asignacion.Clases;

public class Asignacion {

    private int id_asignacion;
    private int id_usuario;
    private int id_inventario;
    private int id_bienes;
    private String bien;
    private String codigo;
    private int id_personal;
    private String personal;
    private int id_gerencia;
    private String gerencia;
    private int id_area;
    private String area;

    public Asignacion() {
    }

    public int getId_asignacion() {
        return id_asignacion;
    }

    public void setId_asignacion(int id_asignacion) {
        this.id_asignacion = id_asignacion;
    }

    public int getId_usuario() {
        return id_usuario;
    }

    public void setId_usuario(int id_usuario) {
        this.id_usuario = id_usuario;
    }

    public int getId_inventario() {
        return id_inventario;
    }

    public void setId_inventario(int id_inventario) {
        this.id_inventario = id_inventario;
    }

    public int getId_bienes() {
        return id_bienes;
    }

    public void setId_bienes(int id_bienes) {
        this.id_bienes = id_bienes;
    }

    public String getBien() {
        return bien;
    }

    public void setBien(String bien) {
        this.bien = bien;
    }

    public String getCodigo() {
        return codigo;
    }

    public void setCodigo(String codigo) {
        this.codigo = codigo;
    }

    public int getId_personal() {
        return id_personal;
    }

    public void setId_personal(int id_personal) {
        this.id_personal = id_personal;
    }

    public String getPersonal() {
        return personal;
    }

    public void setPersonal(String personal) {
        this.personal = personal;
    }

    public int getId_gerencia() {
        return id_gerencia;
    }

    public void setId_gerencia(int id_gerencia) {
        this.id_gerencia = id_gerencia;
    }

    public String getGerencia() {
        return gerencia;
    }

    public void setGerencia(String gerencia) {
        this.gerencia = gerencia;
    }

    public int getId_area() {
        return id_area;
    }

    public void setId_area(int id_area) {
        this.id_area = id_area;
    }

    public String getArea() {
        return area;
    }

    public void setArea(String area) {
        this.area = area;
    }
}
